kimchi/src/circuits/witness/sum_copy_bits_cell.rs

```diff
@@ -0,0 +1,40 @@
+use ark_ff::Field;
+use o1_utils::FieldHelpers;
+
+use super::{variables::Variables, WitnessCell};
+use crate::circuits::polynomial::COLUMNS;
+
+/// Witness cell copied from bits of another witness cell
```

Comment:
This comment is not true.  Suggest renaming to `ConstantBits` cell and updating comments & doc accordingly

Reply:
Actually, I think the way you are using it is .. `VariableBitsCell`

Reply:
Changed it to VariableBitsCell with a "modified" input at calling time instead of in the layout level